index.html

```diff
@@ -63,21 +73,27 @@ <h2 id="sub-main-header">I am a Software Developer</h2>
       </li>
     </ul>
 	</section>
-  <section id="works">
-    <h2 id="works-title">My recent works</h2>
-    <hr>
+  <section id="works" class="flex-column">
+    <div class="flex-row">
+      <h2 id="works-title">My recent works</h2>
+      <div id="hr-desk"></div>
+    </div>
+    <hr id="hr-mob">
     <article id="multipost">
       <img src="./images/multipost.png" alt="Project Image">
-      <h3>Multi-Post Stories</h3>
-      <p>A daily selection of privately personalized reads; no accounts or sign-ups 
-        required. has been the industry's standard dummy text ever since the 1500s, 
-        when an unknown printer took a standard dummy text.</p>
-        <ul class="flex-row">
-          <li>html</li>
-          <li>bootstrap</li>
-          <li>Ruby</li>
-        </ul>
-        <button type="button">See Project</button>
+      <div id="multipost-text" class="flex-column">
```

Comment:
- [ ] Adjust a little the padding/margin on the left side to equal that on the right side.
![Screenshot from 2021-08-20 16-02-24](https://user-images.githubusercontent.com/65501535/130237241-4e3cf647-2efb-40ba-bbcd-934591d6b12d.png)